build.sbt

```diff
@@ -1,8 +1,8 @@
 
 val commonSettings = Seq(
   version            := "0.5",
-  scalaVersion       := "2.12.8",
-  crossScalaVersions := Seq("2.12.8", "2.13.1"),
+  scalaVersion       := "3.0.1",
+  crossScalaVersions := Seq("2.12.13", "2.13.4", "3.0.1"),
```

Comment:
Does it work on the specified versions? If not, please remove 2.x from `crossScalaVersions`.

Reply:
Oh you're right, I wasn't aware `TypeTest` was not available in Scala 2.